From ad89283f9ed36f0f1234c7f005edfc34e8b9fd58 Mon Sep 17 00:00:00 2001
From: Heiko Thiery <heiko.thiery@kontron.com>
Date: Thu, 4 Jun 2020 15:31:56 +0200
Subject: [PATCH] CMakeList.txt: remove version check for faulty libssh version

libssh is fixed by a patch in buildroot so we can drop the libssh
version check for 0.9.3 and 0.9.4.

See: 39099153d9404cd6f107f0fd92ce8686bbd1beb2

Signed-off-by: Heiko Thiery <heiko.thiery@kontron.com>
---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f21fec9..29ef202 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,9 +148,6 @@ endif()
 # dependencies - libssh
 if(ENABLE_SSH)
     find_package(LibSSH 0.7.0 REQUIRED)
-    if(LIBSSH_VERSION VERSION_EQUAL 0.9.3 OR LIBSSH_VERSION VERSION_EQUAL 0.9.4)
-        message(FATAL_ERROR "LibSSH ${LIBSSH_VERSION} includes regression bugs and libnetconf2 will NOT work properly, try to use another version")
-    endif()
 
     target_link_libraries(netconf2 ${LIBSSH_LIBRARIES})
     list(APPEND CMAKE_REQUIRED_LIBRARIES ${LIBSSH_LIBRARIES})
-- 
2.20.1

